web/src/stream_list.ts

```diff
@@ -528,6 +528,8 @@ class StreamSidebarRow {
         this.update_unread_count();
     }
 
+    // TODO: This probably isn't needed anymore, but some test logic
+    // depends on it.
```

Comment:
We're no longer using this classname, but I wasn't sure if it made sense to spend time now removing the classname and rewriting the tests in `stream_list.test.js` that use it to test `row.update_whether_active();`